promql/functions.go

```diff
@@ -1383,54 +1366,25 @@ func funcHistogramStdDev(vals []parser.Value, _ parser.Expressions, enh *EvalNod
 		}
 		enh.Out = append(enh.Out, Sample{
```

Comment:
I'd like to make the same suggestion as for `funcStdvarOverTime`:
```suggestion
		if varianceToResult != nil {
			variance = varianceToResult(variance)
		}
		enh.Out = append(enh.Out, Sample{
```